torch/csrc/api/include/torch/ordered_dict.h

```diff
@@ -221,14 +225,14 @@ class OrderedDict<Key, Value>::Item {
   }
 
   /// Returns a `(key, value)` pair.
-  const std::pair<const Key, Value>& pair() const noexcept {
+  const std::pair<const Key, Value> &pair() const noexcept {
```

Comment:
nit: keeping it `const std::pair<const Key, Value>& pair()` might make it more readable, as it's less common to see `&function_name()` in a function declaration, and can make people confused about the behavior of the function.

Reply:
CI seems to fail with
```
/var/lib/jenkins/workspace/torch/csrc/api/include/torch/ordered_dict.h:229:12: error: returning reference to temporary [-Werror=return-local-addr]
      return pair_;
             ^~~~~
```
I think the error is because `std::pair<const Key, Value>` is a different type from `std::pair<Key, Value>`, hence a copy of `pair_` is created which is a temporary value. To fix this error while making sure we return a const reference to `pair_` in this function, we should likely change the return type to `const std::pair<Key, Value>`.